rollbar-wai/rollbar-wai.cabal

```diff
@@ -54,7 +54,8 @@ library
     , text >=1.2 && <2.1
     , unordered-containers >=0.2 && <1
     , wai >=3.2 && <4
-    , wai-extra >=3.0 && <4
+    , wai-extra >=3.0 && < 3.1.17  
+    -- TODO: update breaks yesod-core, see https://github.com/yesodweb/yesod/issues/1854
```

Comment:
```suggestion
    , wai-extra >=3.0 && < 3.1.17 -- TODO: upper bound added to avoid breakage, remove when this issue is fixed https://github.com/yesodweb/yesod/issues/1854
```

Reply:
I suggested moving the comment to the same line as `wai-extra` (to make sure there is no confusion that it points specifically to that dependency) and added a specific action item for when to remove the TODO.